.github/workflows/activate-docker-distribution.yaml

```diff
@@ -71,16 +71,16 @@ jobs:
       - name: "Update image tag"
         run: |
           set +e
-          docker pull ${{ env.SOURCE_IMAGE_TAG }}
-          TAG=${{ env.DESTINATION_IMAGE_TAG }}
+          docker pull "${{ env.SOURCE_IMAGE }}:${{ inputs.source_image_tag }}"
+          TAG="${{ env.DESTINATION_IMAGE_TAG }}"
```

Comment:
As talked about should probably rename to "${{ env.SOURCE_IMAGE }}:{inputs.destination_image_tag}" for simplicity.

If you want to be more correct you should rename SOURCE_IMAGE to SOURCE_IMAGE_REPO